composer.json

```diff
@@ -7,7 +7,7 @@
         "psr-4": {"Sil\\PhpEnv\\": "src/"}
     },
     "require": {
-        "php": "^7.4"
+        "php": "^8.1"
```

Comment:
Is there a need to drop support for all prior versions of PHP? I'd recommend leaving this as it was, personally.

Reply:
Wouldn't it need to be `^7.4 || ^8.0` or something like that?

Reply:
Oh, true. 👍